bin/extract_to_arb.dart

```diff
@@ -54,6 +53,10 @@ main(List<String> args) {
       defaultsTo: null,
       callback: (value) => locale = value,
       help: 'Specify the locale set inside the arb file.');
+  parser.addFlag("with-source-text",
+      defaultsTo: false,
+      callback: (x) => extraction.includeSourceText = x,
+      help: 'Include source_text in meta information');
```

Comment:
nit: the other help texts end with a period

Reply:
You've got a point there, I'll add it.